registry-index-service/run-registry-index.sh

```diff
@@ -58,6 +58,7 @@ function create_service() {
                 WorkingDirectory=$(pwd)
                 Environment=\"DATA_DIR=$data_dir\"
                 Environment=\"ACTION=recover\"
+                Environment=\"MODE=vm\"
```

Comment:
Can the execution mode be determined based on the index-manager.json file in the bundle? (If it has a suffix o `dev` then it is dev mode else it is beta mode)